src/SingleCellAnnotator.imjoy.html

```diff
@@ -1060,9 +1074,9 @@
         else{
           this.mask_history.push({'add': event.feature})
           if(!event.feature.get('cell_idx') && event.feature.get('cell_idx') !== 0){
-            event.feature.set('label', 'cell')
             this.current_image_data.max_cell_id++      
             event.feature.set('cell_idx', this.current_image_data.max_cell_id)
+            event.feature.set('label', 'cell')
```

Comment:
this is to make the format consistent with the original order, cell first then cell_idx